src/main/java/io/vertx/core/spi/cluster/ClusterManager.java

```diff
@@ -68,7 +60,7 @@
   /**
    * Return the unique node ID for this node
    */
-  String getNodeID();
+  String getNodeId();
```

Comment:
why changing case ?

Reply:
I had `nodeId` vars here and there, so just for consistency. I can revert if it's a biggie.

Reply:
no it's not.